plugin_execution_providers/tensorrt/utils/cuda/cuda_call.h

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+// -----------------------------------------------------------------------
+// Error handling
+// -----------------------------------------------------------------------
+//
+template <typename ERRTYPE>
+const char* CudaErrString(ERRTYPE) {
+  THROW();
+}
+
+template <typename ERRTYPE, bool THRW>
+std::conditional_t<THRW, void, OrtStatus*> CudaCall(
+    ERRTYPE retCode, const char* exprString, const char* libName, ERRTYPE successCode, const char* msg, const char* file, const int line) {
+  if (retCode != successCode) {
+    try {
+      // #ifdef _WIN32
+      // std::string hostname_str = GetEnvironmentVar("COMPUTERNAME");
+      // if (hostname_str.empty()) {
+      // hostname_str = "?";
+      //}
+      // const char* hostname = hostname_str.c_str();
+      // #else
+      // char hostname[HOST_NAME_MAX];
+      // if (gethostname(hostname, HOST_NAME_MAX) != 0)
+      // strcpy(hostname, "?");
+      // #endif
```

Comment:
general: there seems to be quite a lot of commented out code in this PR. it's not ideal because it can easily get out of date. can we avoid adding commented out code?